compiler/src/org.graalvm.compiler.java/src/org/graalvm/compiler/java/BytecodeParser.java

```diff
@@ -5264,8 +5264,10 @@ public final void processBytecode(int bci, int opcode) {
             case IF_ICMPLE      : genIfSame(JavaKind.Int, Condition.LE); break;
             case IF_ACMPEQ      : genIfSame(JavaKind.Object, Condition.EQ); break;
             case IF_ACMPNE      : genIfSame(JavaKind.Object, Condition.NE); break;
-            case GOTO           : genGoto(); break;
-            case JSR            : genJsr(stream.readBranchDest()); break;
+            case GOTO           : // fall through
```

Comment:
This switch statement is ordered by opcode number. I don't see this change as bringing enough value to justify changing the ordering.